Short-Answer/Algorithms_Answers.md

```diff
@@ -2,13 +2,13 @@
 
 ## Exercise I
 
-a)
+a) Time - O(n) Space - O(2)
 
 
-b)
+b) Time - O(n^2) Space - O(3)
```

Comment:
think a little harder on this one. The loops iterate at different rates...so it's not quite O(n^2)....